data/quarg/quarg missions.txt

```diff
@@ -259,3 +259,45 @@ mission "Ask Quarg About Coalition Early"
 			label end
 			`	Before you can ask it anything else, the Quarg bids you farewell, and walks away.`
 				decline
+
+
+
+mission "Quarg Pug Arfecta Warning"
+	landing
+	to offer
+		has "First Contact: Quarg: offered"
+		has "flagship model: Pug Arfecta"
+	source
+		attributes "quarg"
+		attributes "ringworld"
+	on offer
+		conversation
+			`As soon as you land your ship a crowd of visibly irritated Quarg start to gather, and as you go to disembark you find a rather imposing Quarg blocks your exit while staring piercingly at your ship.`
+			`"It is as though a baby in swaddling has tamed a dragon and now rides it through the sky spewing fire with reckless abandon," it says with a mix of amusement and disgust.`
+			choice
+				`	"It's a long story..."`
+					goto evasive
+				`	"I prefer to frame it as an act of righteous defiance."`
+					goto righteous
+				`	"The Pug fancy themselves gods, but God is dead, soo..."`
+					goto anarchy
+				`	"I saw something I liked, so I made it mine."`
+					goto greedy
+			label evasive
+			`	"It matters not how it happened."`
```

Comment:
I think that this line and maybe all others could be missing the usual Quarg elements of visual depictions of their body?
Or at least some more description, like `he scoffs, "it matters not how it happened"`

The Quarg body things happening is valid for all others phrases too I think, the Quarg would not be willing to speak much about it but you should be able to see all kinds of emotions? The same kind we get when we show the broken ring world fragment

Reply:
Sounds good. Will make the change